### ISSUE TITLE
access to the paper

### ISSUE THREAD

**Issue by 9932916355** [huang229/retinal_vascular_segmentation#1]
Hello
How can I access the text of the article, is this article unpublished?
Can I base this article on my research work?

**Comment by yeyuxmf** [comment]
@9932916355 Yes. This is just a semi-finished paper, which is explained in the readme. There is no innovation. It is perfectly fine for you to add your work on top of this for academic research.

**Comment by yeyuxmf** [comment]
I don't want to do this anymore.
